src/style.css

```
@@ -1,3 +1,90 @@
 body {
   margin: 0;
+  padding: 0;
+  font-optical-sizing: auto;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+  font-variant: normal;
+  line-height: 100%;
+  letter-spacing: 0%;
+  text-transform: uppercase;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  width: 100vw; /* Розтягує заголовок на всю ширину вікна браузера */
+  margin: 0; /* Видаляє будь-які зовнішні відступи */
+  padding: 0; /* Видаляє внутрішні відступи, якщо є */
+  box-sizing: border-box; /* Враховує padding і border у ширині */
```

Remove all comments and add padding to header
```suggestion
  width: 100%;
  margin: 0;
  padding: 0 50px;
  box-sizing: border-box;
```